src/gui/src/clockWidget.cpp

```diff
@@ -1544,4 +1615,51 @@ void ClockWidget::fit()
   }
 }
 
+void ClockWidget::findInCts(const Selected& selection)
+{
+  if (!selection) {
+    return;
+  }
+  findInCts(SelectionSet({
+      selection,
+  }));
+}
+
+void ClockWidget::findInCts(const SelectionSet& selections)
+{
+  if (views_.empty()) {
+    return;
+  }
+  if (selections.empty()) {
+    return;
+  }
+
+  std::set<int> found;
+  std::set<int> not_found;
+  int min_found = -1;
+  // ClockTreeView* finded_view;
+  for (int i = 0; i < views_.size(); i++) {
+    int n_found = views_[i]->findInCts(selections);
+
+    if (n_found) {
```

Comment:
Avoid checking integers as if they were bools. Also, these names are not descriptive:
n_found --> `number_of_nodes_found` or `found_nodes_count` ..

Reply:
Fixed.
I changed the function ClockTreeView::findInCts to ClockTreeView::changeSelection.
Now the function returns a boolean indicating if the selection changed.
The variable name was changed to selection_changed.